requirements/production.in

```diff
@@ -0,0 +1,3 @@
+-r base.txt
+
+uwsgi==2.0.17
```

Comment:
Deze versie hoeft ook niet gepint te worden, wellicht zijn er in het verleden problemen geweest maar aangezien we de Python versie en Django versie upgraden lijkt dit me een goed moment om deze ook op te hogen
```suggestion
uwsgi
```